docs/docsite/conf.py

```diff
@@ -8,10 +8,10 @@
 sys.path.insert(0, os.path.abspath('./rst/rest_api/_swagger'))
 
 project = u'Ansible AWX'
-copyright = u'2023, Red Hat'
+copyright = u'2024, Red Hat'
```

Comment:
RH legal said we don't need to have a date in the copyright statement.  So you can pull 2024 out of this and never have to edit it again! Also if this is opensource, we may want a more generic owner. like copyright = "AWX project contributors"

Reply:
Let's do this is in a subsequent PR.